stdlib/public/core/Stride.swift.gyb

```diff
@@ -234,6 +234,23 @@ public struct StrideTo<Element : Strideable> : Sequence, CustomReflectable {
   }
 
   @_inlineable
+  public func _preprocessingPass<R>(
+    _ preprocess: () throws -> R
+  ) rethrows -> R? {
+    return try preprocess()
+  }
+
+  @_inlineable
+  public func _customContainsEquatableElement(
+    _ element: Element
+  ) -> Bool? {
+    if element < _start || _end <= element {
```

Comment:
Is it worth adding an overload for when `Element: FixedWidthInteger` that would compute contains entirely?

Reply:
Hm...